setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='py-chillog',
-    version='0.1',
+    version='0.1.1',
```

Comment:
Can you bump it to `0.2.1`?
I checked in pypi it already on version `0.2`. its my fault not to upgrade version in here too 🙏